test/test_console_tool.py

```diff
@@ -1059,6 +1059,41 @@ def test_sync_dry_run(self):
             ''' % (mtime)
             self._run_command(['list_file_names', 'my-bucket'], expected_stdout, '', 0)
 
+    def test_sync_exclude_all_symlinks(self):
+        self._authorize_account()
+        self._create_my_bucket()
+
+        with TempDir() as temp_dir:
+            file_path = os.path.join(temp_dir, 'test.txt')
+            with open(file_path, 'wb') as f:
```

Comment:
you should use `_make_local_file()` here

Reply:
Fixed.